src/runtime/pkg/katautils/config.go

```diff
@@ -14,7 +14,7 @@ import (
 	"strings"
 
 	"github.com/BurntSushi/toml"
-	govmmQemu "github.com/intel/govmm/qemu"
+	govmmQemu "github.com/kata-containers/govmm/qemu"
```

Comment:
\o/ :smile:

Can you change the revendor commit to add a `Fixes: #859` comment please.

Reply:
Of course. Some reason that I created this PR at this time is helping CI to do the test for me :-)